concepticondata/conceptlists/Koelle-1854-335.tsv

```diff
@@ -0,0 +1,336 @@
+Koelle-1854-335-121	121	Heaven (sky)	1565	HEAVEN
+Koelle-1854-335-122	122	Heel	980	HEEL
+Koelle-1854-335-123	123	Hell	878	HELL
+Koelle-1854-335-124	124	Hoe	284	HOE
+Koelle-1854-335-125	125	Honey	942	HONEY
+Koelle-1854-335-126	126	Horse	615	HORSE
+Koelle-1854-335-127	127	Hot	1286	HOT
+Koelle-1854-335-128	128	House	1252	HOUSE
+Koelle-1854-335-129	129	I bathe (wash myself)	138	BATHE
+Koelle-1854-335-130	130	I beg	3534	BEG
+Koelle-1854-335-131	131	I break a stick	2558	BREAK (CLEAVE)
+Koelle-1854-335-132	132	I breathe	1407	BREATHE
+Koelle-1854-335-133	133	I buy	1869	BUY
+Koelle-1854-335-134	134	I call a slave		
+Koelle-1854-335-135	135	I catch a fish	2638	FISHING
+Koelle-1854-335-136	136	I come	1446	COME
+Koelle-1854-335-137	137	I cook meat		
+Koelle-1854-335-138	138	I cough	879	COUGH
+Koelle-1854-335-139	139	I cover a pot		
+Koelle-1854-335-140	140	I cut a tree	3151	CUT (WITH AXE)
+Koelle-1854-335-141	141	I dance	1879	DANCE
+Koelle-1854-335-142	142	I die	1494	DIE
+Koelle-1854-335-143	143	Idol	1945	IDOL
+Koelle-1854-335-144	144	I do not dance		
+Koelle-1854-335-145	145	I do not play		
+Koelle-1854-335-146	146	I dream	1920	DREAM (SOMETHING)
+Koelle-1854-335-147	147	I drink water		
+Koelle-1854-335-148	148	I eat corn		
+Koelle-1854-335-149	149	I eat rice		
+Koelle-1854-335-150	150	I eat yam		
+Koelle-1854-335-151	151	I eat yams		
+Koelle-1854-335-152	152	I fall	2894	TUMBLE (FALL DOWN)
+Koelle-1854-335-153	153	I flog a child		
+Koelle-1854-335-154	154	I give thee	1447	GIVE
+Koelle-1854-335-155	155	I go	695	GO
+Koelle-1854-335-156	156	I hear	1408	HEAR
+Koelle-1854-335-157	157	I kill a fowl		
+Koelle-1854-335-158	158	I kneel	66	KNEEL
+Koelle-1854-335-159	159	I laugh	1355	LAUGH
+Koelle-1854-335-160	160	I lie down	215	LIE DOWN
+Koelle-1854-335-161	161	I love thee	923	LOVE
+Koelle-1854-335-162	162	Ink	2643	INK
+Koelle-1854-335-163	163	Inner Hand	1183	PALM OF HAND
+Koelle-1854-335-164	164	I play	1413	PLAY
+Koelle-1854-335-165	165	I pray to god (beg god)	24	PRAY
+Koelle-1854-335-166	166	I rise	3207	GO UP OR RISE (ONESELF)
+Koelle-1854-335-167	167	Iron	621	IRON
+Koelle-1854-335-168	168	I run	1519	RUN
+Koelle-1854-335-169	169	I see	1409	SEE
+Koelle-1854-335-170	170	I sell	1571	SELL
+Koelle-1854-335-171	171	I sew	1457	SEW
+Koelle-1854-335-172	172	I sew a shirt		
+Koelle-1854-335-173	173	I sit down	1649	SIT DOWN
+Koelle-1854-335-174	174	I sleep	1585	SLEEP
+Koelle-1854-335-175	175	I sneeze	1621	SNEEZE
+Koelle-1854-335-176	176	I snore	1983	SNORE
+Koelle-1854-335-177	177	I speak	1623	SPEAK
+Koelle-1854-335-178	178	I stop	2880	HALT (STOP)
+Koelle-1854-335-179	179	I take	1749	TAKE
+Koelle-1854-335-180	180	Itch	148	ITCH
+Koelle-1854-335-181	181	Ivory		
+Koelle-1854-335-182	182	I weep	1839	CRY
+Koelle-1854-335-183	183	King	1508	KING
+Koelle-1854-335-184	184	Knee	1371	KNEE
+Koelle-1854-335-185	185	Knife	1352	KNIFE
+Koelle-1854-335-186	186	Kuskus (bearing like Oats)		
+Koelle-1854-335-187	187	Leaf	628	LEAF
+Koelle-1854-335-188	188	Leaf (grass)	628	LEAF
```

Comment:
LEAF is mapped twice.

Reply:
Same case for EWE and FIREWOOD. Once more, if needed we can drop the mapping for 188 (in this case, I think there is only one language using it).